src/sentry/flags/docs/api.md

```diff
@@ -99,6 +118,14 @@ Requests from web hook providers can be signed. We use the signing secret to ver
 
 - Response 201
 
+## Signing Secret [/organizations/<organization_id_or_slug>/flags/hooks/signing-secrets/<signing_secret_id>/]
```

Comment:
how do we get the signing secret id?

Reply:
I'll return the id in the list response.  Updating now.